public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -517,7 +575,15 @@ export class PanelChrome extends PureComponent<Props, State> {
       [`panel-alert-state--${alertState}`]: alertState !== undefined,
     });
 
-    return (
+    // for new panel header design
+    const onCancelQuery = () => panel.getQueryRunner().cancelQuery();
+    const title = panel.getDisplayTitle();
+    const noPadding: PanelPadding = plugin.noPadding ? 'none' : 'md';
+    const leftItems = [
+      <PanelHeaderLoadingIndicator state={data.state} onClick={onCancelQuery} key="loading-indicator" />,
+    ];
+
+    return !config.featureToggles.newPanelChromeUI ? (
```

Comment:
think this would be easier to read if it was just 

```ts
if (config.featureToggles.newPanelChromeUI) {
 return (
 )
}

the huge multi line  ? return is usually harder to see what is part of which condition, at least for me :)